text/0000-rfc-process.md

```diff
@@ -0,0 +1,109 @@
+# Meta
+[meta]: #meta
+- Name: RFC Process
+- Start Date: 2019-04-09
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: N/A
+
+# Summary
+[summary]: #summary
+
+The "RFC" (request for comments) process provide a consistent and controlled path for new features to enter the spec, libraries, or tools, so that all stakeholders can be confident about the direction Cloud Native Buildpacks is evolving in.
+
+# Motivation
+[motivation]: #motivation
+
+Now that we've entered the beta and people are trying out Cloud Native Buildpacks in the wild, we need to take more care in introducing change. As part of road of incubation, we need to demonstrate maturity and stability. This process documents a path for anyone to contribute and have input on features.
+
+# What it is
+[what-it-is]: #what-it-is
+
+RFCs record potential change and with the context and information at the given time. This process provides a defined process for anyone wishing to contribute to the Cloud Native Buildpacks project as well as collect a diverse set of inputs give opportunity for engagement. Anyone who chooses not to actively partcipate in any RFC is presumed to trust their colleagues on the matter. Once a RFC is accepted, they can be referenced as read-only documents in this repository until replaced or ammended by another RFC when context has significantly changed.
+
+# How it Works
+[how-it-works]: #how-it-works
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the community and the core team / respective subteam (team).
+
+## What's in Scope
+
+You'll need to follow this process for anything considered "substantial". The things that we consider "substantial" will evolve over time, but they will generally be any user facing change. The Cloud Native Buildpacks project currently has users in 5 types:
+
+- Buildpack Users - these are developers who want to go from source code to container image
+- Buildpack Authors - these are developers who are building the buildpacks
+- Platform Operator - these are the operators who will be running the buildpack infrastructure
+- Platform Implementor - these are developers who will be implementing the buildpack infrastructure
+- Project Contributor - people contributing to the Cloud Native Buildpacks project
+
+What constitutes a "substantial" change may include the following:
+
+- changes to the spec (buildpack author, platform operator)
+- adding/removing/changing pack commands (buildpack user, buildpack author)
+- lifecycle library interface (platform implementor)
+- process changes (project contributor)
+- governance (project contributor)
+
+If you submit a pull request the team deems warrants a RFC will politely closed with a request for one to be opened first.
```

Comment:
```suggestion
If you submit a pull request the team deems warrants an RFC, it will be politely closed with a request for an RFC to be opened first.
```